tensorbay/client/version.py

```diff
@@ -63,8 +63,10 @@ def _commit(self, title: str, description: str, tag: Optional[str] = None) -> st
         response = self._client.open_api_do("POST", "commits", self._dataset_id, json=post_data)
         return response.json()["commitId"]  # type: ignore[no-any-return]
 
-    def _generate_drafts(self, offset: int = 0, limit: int = 128) -> Generator[Draft, None, int]:
-        params = {"offset": offset, "limit": limit}
+    def _generate_drafts(
```

Comment:
should we add integration test about this pr in `tests.test_draft.test_list_drafts`

Reply:
Also do we need to update the doc about this feature?